aten/src/ATen/native/cuda/IndexKernel.cu

```diff
@@ -5,15 +5,67 @@
 #include <ATen/native/TensorIterator.h>
 #include <ATen/core/Array.h>
 #include <ATen/cuda/detail/OffsetCalculator.cuh>
+#include <ATen/cuda/detail/IndexUtils.cuh>
 #include <ATen/cuda/CUDAContext.h>
+#include <ATen/cuda/CUDAApplyUtils.cuh>
 #include <ATen/ExpandUtils.h>
+#include <THC/THCTensorInfo.cuh>
```

Comment:
I included this header to get the MAX_CUTORCH_DIMS and CUTORCH_DIM_WARNING macros from the TH api. Is there a corresponding ATen macro for me to use / better to hardcode them here?